packages/docs/docusaurus.config.js

```diff
@@ -134,6 +134,25 @@ const config = {
         theme: lightCodeTheme,
         darkTheme: darkCodeTheme,
       },
+      algolia: {
+        // The application ID provided by Algolia
+        appId: 'LRQC6J0BO6',
+        // Public API key: it is safe to commit it
+        apiKey: 'ab28c5ea429f2a16f7360d8a55a25f88',
```

Comment:
Someone isn't going to be able to steal this and use it? Or is it all free anyway?